docs/SRS/SRS.tex

```diff
@@ -487,25 +488,50 @@ \subsubsection{Work Orders}
 \section{Non-functional Requirements}
 
 \subsection{Look and Feel Requirements}
+\begin{enumerate}
+	\item [{LF}1.] The system shall adjust and scale to fit the physical screen size
+	\item [{LF}2.] The system shall have font and colours that will allow users to easily read the text
+\end{enumerate}
 
 \subsection{Usability and Humanity Requirements}
+\begin{enumerate}
+	\item [{UH}1.] The system shall be easy for any person to use all features
+	\item [{UH}2.] The system shall accessible by any desktop or mobile device and any operating system
+\end{enumerate}
 
 \subsection{Performance Requirements}
+\begin{enumerate}
+	\item [{PR}1.] The system shall respond to the user's interactions within 0.5 seconds
+	\item [{PR}2.] The system shall display dollar amounts rounded to 2 decimal places
+	\item [{PR}3.] The system shall be developed in a way where it is easy to scale and add additional features in the future
+\end{enumerate}
 
 \subsection{Operational and Environmental Requirements}
+\begin{enumerate}
+	\item [{OE}1.] The system shall be able to operate on desktops and mobile devices
+	\item [{OE}2.] The system shall be developed following the selected coding standards and cosistent methodology
```

Comment:
I don't know if this is a requirement. It doesn't have anything to do with what the system will do.